doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -321,6 +309,119 @@ Error list (HTTP response code in paranthesis):
 * UnsupportedMediaType (415). Request content type is not supported.
 * InvalidModification (422). Some piece of information is missing in payload.
 
+## Geospatial properties of entities
+
+The geospatial properties of a context entity can be represented by means of regular context attributes.
+The provision of geospatial properties enables the resolution of geographical queries.
+
+Two different syntaxes must be supported by compliant implementations: 
+
+* *Simple Location Format*. It is meant as a very lightweight format that developers and users can quickly
+and easily add to their existing entities with little effort.
+
+* *GeoJSON*.  [GeoJSON](https://tools.ietf.org/html/draft-butler-geojson-06)
+is a geospatial data interchange format based on the JavaScript Object Notation (JSON). GeoJSON provides greater flexibility
+allowing the representation of complex geospatial features.
+
+Additionally, geospatial properties might be declared by means of [civic addresses](https://schema.org/PostalAddress),
```

Comment:
Maybe this paragraph should be removed... It is not actually adding any "normative" aspect to the specification and its vageness could be confusing.